agent/config/config.go

```diff
@@ -92,7 +92,7 @@ const (
 
 	// minimumTaskCleanupWaitDuration specifies the minimum duration to wait before cleaning up
 	// a task's container. This is used to enforce sane values for the config.TaskCleanupWaitDuration field.
-	minimumTaskCleanupWaitDuration = 1 * time.Minute
+	minimumTaskCleanupWaitDuration = time.Second
```

Comment:
LGTM, curious though why exactly this is in place, and are there any possible issues from not waiting long enough?

Reply:
I think it was an entirely arbitrary decision. I tested various scenarios and `1s` seems to work fine. 
At any rate, this is just the minimum allowed value and it will only be used if customer deems it useful.